.github/workflows/node.js.yml

```diff
@@ -25,3 +25,4 @@ jobs:
       run: |
         npm ci
         npm test
+        npm run lint
```

Comment:
Huge +1 to this! Both that it be a separate command, not bundled under `test`, and that it be run after the unit tests.